### PR TITLE
[PGE-178206224] Fix Prebid 5.0 compliance: leverage floors module

### PR DESCRIPTION
**Pull request by maphe**
https://districtm.atlassian.net/browse/PGE-178206224

### CODE REVIEW

File: modules/sharethroughBidAdapter.js [hunk 1]
```diff
@@ -57,8 +57,9 @@ export const sharethroughAdapterSpec = {
         query.schain = JSON.stringify(bidRequest.schain);
       }
 
-      if (bidRequest.bidfloor) {
-        query.bidfloor = parseFloat(bidRequest.bidfloor);
+      const floor = getFloor(bidRequest);
+      if (floor) {
```
Comment (javadreamer) [inline modules/sharethroughBidAdapter.js:61]: @maphe - if the floor were zero, would this evaluate to false?

Comment (maphe) [reply modules/sharethroughBidAdapter.js:61]: yes, is there value in sending `0` to the ad server rather than no floor at all?

Comment (maphe) [reply modules/sharethroughBidAdapter.js:61]: confirmed with STX, they are the same

**Review by javadreamer — APPROVED** [review]

**Comment by maphe** [comment]
Official PR: prebid/Prebid.js#6874

**Pull request closed without merging**